team-config.yml

```diff
@@ -0,0 +1,9 @@
+sds-platform:
+  team: "platform operations"
+  namespace: ""
+  slack:
+    contact_channel: "#platops-help"
+    build_notices_channel: "#platops-build-notices"
+  tags:
+    application: "core"
```

Comment:
application and business area should be explained somehow, maybe a comment at the top of the file saying businessArea must be one of ...

and that application is normally the team name except for 'platform' which is core